clang/test/SemaSYCL/spurious-host-warning-ReqdSubGroupSize.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-host -fsyntax-only -verify %s -DSYCLHOST
+// RUN: %clang_cc1 -fsyntax-only -verify %s
+
+#ifdef SYCLHOST
```

Comment:
There is only 1 invocation, but 2 different SYCL_HOST conditions, why the difference?  This test doesnt seem complete.

Reply:
Sorry for the wrong test. I have updated reqd-sub-group-size.cpp to add host test.